src/components/my-history/custom-my-history.vue

```diff
@@ -62,15 +62,17 @@
         </div>
```

Comment:
Should the close buttons be aligned with the switch?

<img width="495" alt="image" src="https://user-images.githubusercontent.com/75546736/213674057-40467265-706f-462e-88c7-8de4de30a89c.png">